lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/client/MultiRuntimeConfigurationBuilder.java

```diff
@@ -123,7 +125,7 @@ public Configuration build() {
       this.runtimeNames.add(this.defaultRuntime.get());
     }
 
-    ConfigurationModuleBuilder conf = new MultiRuntimeHelperConfiguration();
+    JavaConfigurationBuilder conf = Tang.Factory.getTang().newConfigurationBuilder();
```

Comment:
The MultiRuntimeHelperConfiguration was actually meaningless. I used it to create an empty configuration builder as I wasn't aware of this way for creating empty conf builder.